app/backend/hiStoryBackend/api/views/v1/memory_post_views.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.views import APIView
+
+from api.helpers import json_response_helpers as jrh
+from api.models import MemoryPost
+from api.serializers import MemoryPostSerializer
+
+class MemoryPostView(APIView):
+
+	def post(self, request, format = None):
+		memory_post_serializer = MemoryPostSerializer(data = request.data)
+		if memory_post_serializer.is_valid():
+			memory_post = memory_post_serializer.save(user = request.user, story = request.data.getlist('story[]', []))
```

Comment:
No need for assigning the result of `memory_post_serializer.save()` to `memory_post` variable. It is not used. Also, you can remove the import statement for `MemoryPost` above.